### PR TITLE
PSP-8573 | Updated list headers

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/frontend/src/features/acquisition/list/AcquisitionListView.tsx [hunk 1]
```diff
@@ -95,7 +96,7 @@ export const AcquisitionListView: React.FunctionComponent<
   return (
     <Styled.ListPage>
       <Styled.Scrollable>
-        <Styled.PageHeader>Acquisition Files</Styled.PageHeader>
+        <CommonStyled.H1>Acquistion Files</CommonStyled.H1>
```
Comment (asanchezr) [inline source/frontend/src/features/acquisition/list/AcquisitionListView.tsx:99]: typo: Acquistion -> Acquisition

File: source/frontend/src/features/research/list/ResearchListView.tsx [hunk 2]
```diff
@@ -64,7 +65,7 @@ export const ResearchListView: React.FunctionComponent<React.PropsWithChildren<u
   return (
     <Styled.ListPage>
       <Styled.Scrollable>
-        <Styled.PageHeader>Research Files</Styled.PageHeader>
+        <CommonStyled.H1>Research Files</CommonStyled.H1>
```
Comment (asanchezr) [inline source/frontend/src/features/research/list/ResearchListView.tsx:68]: The story in JIRA calls for adding the corresponding icons to each "manage" page but I see no icons here. Was this missed from implementation or did the story requirements changed?

image

Comment (FuriousLlama) [reply source/frontend/src/features/research/list/ResearchListView.tsx]: good catch. I've updated the list view titles with the icons

**Review by asanchezr — APPROVED** [review]
Approved but you need to fix the tests

**Pull request merged**